libraries/AP_Mount/AP_Mount.h

```diff
@@ -19,6 +19,19 @@
 ************************************************************/
 #pragma once
 
+#include <AP_HAL/AP_HAL.h>
+#include <AP_AHRS/AP_AHRS.h>
+
+#ifndef HAL_MOUNT_ENABLED
+#define HAL_MOUNT_ENABLED !HAL_MINIMIZE_FEATURES
```

Comment:
Just my opinion here, but I find it more convenient to use vehicle define.h to select which feature we want instead of digging into each library to enable or disable it.

Reply:
Yeah, but we are going in a different direction to this